test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1181,6 +1201,33 @@ static void ipsec_test_capability(void)
 	CU_ASSERT(odp_ipsec_capability(&capa) == 0);
 }
 
+static void test_ipsec_stats(void)
+{
+	ipsec_test_flags flags = {0};
+	uint32_t a, c;
+	int rc = -1;
+
+	flags.stats_success = 1;
+
+	for (c = 0; c < ARRAY_SIZE(ciphers); c++)
+		for (a = 0; a < ARRAY_SIZE(auths); a++) {
+			rc = test_esp_out_in(&ciphers[c], &auths[a], &flags);
+			if (rc == 0)
```

Comment:
Is the intent here that stats are checked only for the first supported algorithm combo and not the others? Why? Would it not be much simpler to just check the stats as part of test_esp_out_in_all()?

Reply:
That is correct. We can technically check success stats for all combination in test_esp_out_in_all() but error cases are the trickier ones.